design/ceph/object/store.md

```diff
@@ -792,3 +792,31 @@ Rook will also not perform validation to check if CephObjectStore configs confli
 `config` options. Users should refrain from setting `client.rook-ceph-rgw-*` configs in the
 CephCluster when this feature is used. Documentation should note this for users. Rook can revisit
 this decision in the future to add cross-CRD validation if it becomes a strong user desire/need.
+
+### RGW Operations Logging
```

Comment:
Instead of this design doc, please add this new documentation to ceph-object-store-crd.md.